functions/interface.php

```diff
@@ -107,7 +106,7 @@ function admin_notices() {
 
 				<?php foreach ( $notices['server_config'] as $notice ) : ?>
 
-					<li><?php echo wp_kses_data( $notice ); ?></li>
+					<li><?php echo print_whitelist_html( $notice ); ?></li>
```

Comment:
Yes, ideally I don't wanna do that - because it because harder to track and make sure that all the strings are properly escaped - grrr :( I wonder if I can save strings together with an HTML list of allowed elements and then extract that data just before running `print_whitelist_html()` is that a better approach?